automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeDataprocSpec.scala

```diff
@@ -36,8 +36,7 @@ class RuntimeDataprocSpec
     extends GPAllocFixtureSpec
     with ParallelTestExecution
     with LeonardoTestUtils
-    with NotebookTestUtils
-    with GPAllocBeforeAndAfterAll {
```

Comment:
@Qi77Qi this `GPAllocBeforeAndAfterAll` may have been causing issues in `RuntimePatchSpec` -- this would cause it to release the gpalloc project as soon as `RuntimeDataprocSpec` finishes, which would delete all Dataproc clusters even if `RuntimePatchSpec` is still running I think.

Reply:
oops..that was accidental